classes/blocks/class-carouselheader.php

```diff
@@ -115,6 +101,22 @@ public function prepare_data( $fields ): array {
 		return [];
 	}
 
+	/**
+	 * Load additional frontend assets
+	 */
+	public static function enqueue_frontend_assets() {
+		if ( BlockList::has_block( 'planet4-blocks/carousel-header' ) ) {
```

Comment:
The Gallery block has the same issue

Reply:
Thanks :) fixed too  :wrench:

Reply:
Ah yes good point :eyes:

Reply:
I see for the Carousel Header we should also use it to register the block, the Gallery block already does that :)